scripts/zones/VeLugannon_Palace/mobs/Zipacna.lua

```diff
@@ -4,122 +4,355 @@
 -----------------------------------
 ---@type TMobEntity
 local entity = {}
+local ID = zones[xi.zone.VELUGANNON_PALACE]
+
+-- Spawn points from nm_spawn_points.sql
+local spawnPoints =
+{
+    blueShort   = {x = -196, y = 0,  z = 389},
+    blueLong    = {x = -121, y = 16, z = 420},
+    yellowShort = {x =  191, y = 0,  z = 399},
+    yellowLong  = {x =  141, y = 16, z = 440},
+}
+
+local direction =
+{
+    TO_EAST = 0,
+    TO_WEST = 1,
+    CURRENT = 0,
+}
```

Comment:
Your usage here is logically unsafe. You're using TO_EAST/TO_WEST as enums for comparison, but later on you're using and changing CURRENT. Even though Lua doesn't have a dedicated const/non-mutable enum type, that's what should be implied by this usage.

Consider this:
```lua
-- This shouldn't be changed at runtime, used like an enum
local pathingDirection =
{
    TO_EAST = 0,
    TO_WEST = 1,
}

-- Do whatever you like with this
local currentDirection = pathingDirection.TO_EAST
```

Reply:
You could even enforce this (at the CI level) with LLS annotations: https://luals.github.io/wiki/annotations/#enum

Reply:
Fixed